sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.institutions.controller.ts

```diff
@@ -230,12 +232,26 @@ export class ScholasticStandingInstitutionsController extends BaseController {
     // Validate the text models.
     const textValidations =
       this.applicationWithdrawalImportTextService.validateTextModels(
-        textModels,
+        withdrawalFileData.applicationWithdrawalModels,
       );
     // Assert successful validation.
     this.assertTextValidationsAreValid(textValidations);
-    // TODO Add business validation for application bulk withdrawal.
-
+    const validationModels =
+      await this.applicationWithdrawalImportService.generateApplicationBulkWithdrawalValidationModels(
+        textValidations,
+        withdrawalFileData.header,
+        userToken.authorizations.institutionId,
+        userToken.authorizations.adminLocationsIds,
```

Comment:
Why should we worry about location ids when we have only institution administrators going to do the upload?

![image](https://github.com/bcgov/SIMS/assets/54600590/14823149-e218-46a2-a3b8-816762e26d1b)

Reply:
removed it.